### PR TITLE
Include span information in 'unknown identifier' errors.

### PR DESCRIPTION
**Pull request by jimblandy**

### CODE REVIEW

File: src/front/wgsl/mod.rs [hunk 1]
```diff
@@ -175,6 +175,11 @@ impl<'a> Error<'a> {
                 labels: vec![(accessor_span.clone(), "invalid accessor".to_string())],
                 notes: vec![],
             },
+            Error::UnknownIdent(ref ident_span, ident) => ParseError {
+                message: format!("unknown identifier: '{}'", ident),
+                labels: vec![(ident_span.clone(), "unknown identifier".to_string())],
```
Comment (kvark) [inline src/front/wgsl/mod.rs:180]: I wonder if the second element in this tuple should just be `&'static str`

**Review by kvark — APPROVED** [review]
Looks great!

File: src/front/wgsl/mod.rs (hunk 1 above)
Comment (grovesNL) [reply src/front/wgsl/mod.rs:180]: I think we want to be able to use `String` there too, e.g. we have some `format!`s above

Comment (kvark) [reply src/front/wgsl/mod.rs:180]: there is quite a bit of repetition there between `message` and the second part of the label

**Pull request merged**